specification/introduction/introduction.tex

```diff
@@ -3,59 +3,78 @@ \chapter{Introduction}\label{sec:introduction}
 UAVCAN is a lightweight protocol designed to provide a highly reliable communication method for
 aerospace and robotic applications via robust vehicle bus networks.
 
-This is a non-normative section covering the basic concepts that govern development and maintenance of the specification.
+This is a non-normative section covering the basic concepts that govern development and maintenance of
+the specification.
 
 \section{Design principles}
 
-UAVCAN is designed to adhere to the following set of basic principles.
-
 \begin{description}
     \item[Democratic network] --- There will be no master node.
     All nodes in the network will have the same communication rights; there must be no single point of failure.
 
-    \item[Nodes can exchange long payloads] --- Nodes must be provided with a simple way of exchanging large
-    data structures that cannot fit into a single transport frame (such as GNSS solutions, 3D vectors, etc.).
-    UAVCAN should perform automatic transfer decomposition and reassembly at the protocol level,
+    \item[High-level communication abstractions] --- The protocol will support publish/subscribe and remote procedure
+    call communication semantics with statically defined and statically verified data types (schema).
+    The data types used for communication will be defined in a clear, platform-agnostic way
+    that can be easily understood by machines, including humans.
```

Comment:
did you mean to call humans "machines" in this sentence?

Reply:
I should probably fix this otherwise some adepts of free will might come after me. For now, let's pretend this never happened.